spiketoolkit/postprocessing/postprocessing_tools.py

```diff
@@ -902,14 +904,29 @@ def export_to_phy(recording, sorting, output_folder, n_comp=3, electrode_dimensi
         print('Run:\n\nphy template-gui ', str(output_folder / 'params.py'))
 
 
-def _compute_templates_similarity(templates):
+def _compute_templates_similarity(templates, template_ind):
```

Comment:
Note not related to PR:
This is an important function that should not be hidden and private.

Reply:
There is another or for that